schedule/yast/autoyast/autoyast_lvm.yaml

```diff
@@ -18,8 +17,8 @@ schedule:
   - autoyast/login
   - console/validate_lvm
   - autoyast/verify_lvm_partitions
+  - autoyast/verify_system_lvm
```

Comment:
I'm not 100% sure about naming, as it's for this particular profile and not for generic lvm installation. I have same concern for the ticket I currently work on and I will try to use atomic modules, instead of just moving everything to the single test module.
Also, in this module we do not validate `lvm` at all, so name is definitely misleading.

Reply:
I can separate the modules. Maybe some can be reused in both autoyast_lvm and autoyast_tftp. Maybe we can sync on this.

Reply:
There is only the password check used by tftp test.